skyportal/tests/api/test_allocations.py

```diff
@@ -0,0 +1,97 @@
+from skyportal.tests import api
+
+
+def test_super_user_post_allocation(sedm, public_group, super_admin_token):
```

Comment:
Can we also check some of the cases you're handling regarding posting invalid instrument/group IDs?

Reply:
done